libmscore/score.h

```diff
@@ -541,7 +542,10 @@ class Score : public QObject, public ScoreElement {
       void addMeasure(MeasureBase*, MeasureBase*);
       void readStaff(XmlReader&);
       bool read(XmlReader&);
-
+      void setAnnotation()          { _isAnnotation = 1 ; }
```

Comment:
Yes, this is just a temporary workaround to get the hard coded box to draw
as it was discussed on IRC. This most probably wouldn't persist in the
later implementation.

On Mon, Jun 27, 2016 at 10:53 PM, Marc Sabatella <notifications@github.com>
wrote:

> In libmscore/score.h
> <https://github.com/musescore/MuseScore/pull/2687#discussion_r68618454>:
>
> > @@ -541,7 +542,10 @@ class Score : public QObject, public ScoreElement {
> >        void addMeasure(MeasureBase*, MeasureBase*);
> >        void readStaff(XmlReader&);
> >        bool read(XmlReader&);
> > -
> > +      void setAnnotation()          { _isAnnotation = 1 ; }
>
> This shouldn't be needed. Elements already have a type field, this is what
> the "is" functions test. See my comment above.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2687/files/e60036e9f3aed60ed10dc957d70ce686d2fd5ab8..dc3160283fc38d471f00e2b81461442d486aa395#r68618454>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGf7yteAj5A0W4xP7yaTBfMYlR_8TmG9ks5qQAcjgaJpZM4I7zZq>
> .
>



-- 
Ruchit Agrawal.
IIIT Hyderabad.